open_city_profile/graphene.py

```diff
@@ -1,16 +1,15 @@
 import uuid
-from collections import OrderedDict
+from functools import partial
 
 import graphene
-import graphql
-from django.conf import settings
 from django.forms import MultipleChoiceField
-from django.utils import translation
 from django_filters import MultipleChoiceFilter
+from graphene_django import DjangoObjectType
 from graphene_django.forms.converter import convert_form_field
+from graphene_django.types import ALL_FIELDS
 from parler.models import TranslatableModel
-from parler.utils.context import switch_language
 
+from open_city_profile import settings
```

Comment:
Why isn't `settings` imported from `django.conf`, as is customary?

Reply:
Seems like autoimport fail. I'll fix it.